### PR TITLE
Flow extension improvements

### PR DESCRIPTION
**Pull request by vimtor**
## Description

- Add a new command `Next Session` which is a shortcut for `Skip Session` + `Start Session`
- Make `Reset Timer` start timer (here I'm not sure if we should add a different command like `Restart Timer`)

## Type of change

- Bug fix / improvement for my extension

## Screencast

https://user-images.githubusercontent.com/36263538/152259972-951051db-8d38-49d6-b3af-3dddf687f037.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
I'm not a regular flow user, so I don't know, but is `Next Session` supposed to start the “Break”, I thought it would go to next “Session” 😄 

I think it's fine that `Reset Timer` resets the counter and counting down, if you're unsure, you could add a preference and let the user select it (not required)

File: extensions/flow/src/next-session.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { showHUD, showToast, ToastStyle } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+import { isFlowInstalled } from "./utils";
+
+export default async function skipSession() {
+  if (!(await isFlowInstalled())) {
+    await showToast(ToastStyle.Failure, "Flow is not installed", "https://flowapp.info/");
```
Comment (pernielsentikaer) [inline extensions/flow/src/next-session.ts:7]:
```suggestion
    await showToast(ToastStyle.Failure, "Flow is not installed", "Install it from: https://flowapp.info/");
```
Make the UX a bit better, could you maybe add it everywhere it makes sense

**Comment by vimtor** [comment]
Done! I did the following:
-  Make the `Next Session` truly go to the next extension
- Added more contextual information to error messages
- Decided to keep the reset as reset the pause since I think it is the common action and adding a preference seems unnecessary

File: extensions/flow/src/next-session.ts (hunk 1 above)
Comment (vimtor) [reply extensions/flow/src/next-session.ts]: Done

**Comment by pernielsentikaer** [comment]
Hi @vimtor 

Awesome corrections, it's a killer 🔥 

In this test run, a few times it took like 600ms to execute the command, and I was unsure if I got the timer started – we could consider adding a loading toast – what do you think about that?

**Comment by vimtor** [comment]
> Hi @vimtor
> 
> Awesome corrections, it's a killer 🔥
> 
> In this test run, a few times it took like 600ms to execute the command, and I was unsure if I got the timer started – we could consider adding a loading toast – what do you think about that?

Would this mean that Raycast does not close after the command? To me it seems odd if I see and toast spinner but then I don't see the confirmation one. Can you elaborate on how it should behave?

**Comment by pernielsentikaer** [comment]
> > Hi @vimtor
> > Awesome corrections, it's a killer 🔥
> > In this test run, a few times it took like 600ms to execute the command, and I was unsure if I got the timer started – we could consider adding a loading toast – what do you think about that?
> 
> Would this mean that Raycast does not close after the command? To me it seems odd if I see and toast spinner but then I don't see the confirmation one. Can you elaborate on how it should behave?

Maybe it just a crazy though 😄 Let me show you with good ol' video 💪 

#### Before

https://user-images.githubusercontent.com/38033107/153136929-b53fed99-9fd2-441e-b8fc-c2251ec1f7f1.mp4

### After

https://user-images.githubusercontent.com/38033107/153137031-adee005a-750c-426c-afd6-01e946e3fe73.mp4

**Comment by vimtor** [comment]
I like it but maybe I would keep the extension open and show a success toast

**Comment by pernielsentikaer** [comment]
> I like it but maybe I would keep the extension open and show a success toast

Wouldn't that be a bit anoying to keep it open? I'm not the right to ask since I don't use it 😄 I just feel like we should do something since there is the "huge" lag where the user don't know if anything is wrong or forgot to click enter 😄

**Comment by vimtor** [comment]
Done!

**Review by pernielsentikaer — APPROVED** [review]
Awesome update @vimtor 💪 

Let's send it to final review 💯

**Pull request merged**